cli/lib/kontena/cli/master/ssh_command.rb

```diff
@@ -0,0 +1,43 @@
+module Kontena::Cli::Master
+  class SshCommand < Kontena::Command
+
+    include Kontena::Cli::Common
+
+    parameter "[COMMANDS] ...", "Run command on host"
+
+    option ["-i", "--identity-file"], "IDENTITY_FILE", "Path to ssh private key"
+    option ["-u", "--user"], "USER", "Login as a user", default: "core"
+
+    requires_current_master
+
+    def master_host
+      require 'uri'
+      URI.parse(current_master.url).host
+    end
+
+    def master_provider
+      Kontena.run('master config get --return server.provider', returning: :result)
```

Comment:
Repeating myself but there is something here that I don't like (related to return)... but let's discuss this more in a separate issue/pr.